pubspec.yaml

```diff
@@ -42,7 +42,7 @@ dependencies:
   shared_preferences: ^2.0.3
   sqflite: ^2.0.0+3
   webview_flutter: ^1.0.7
-  youtube_explode_dart: ^1.9.6
+  youtube_explode_dart: ^1.9.10
```

Comment:
The latest version seems to be ^1.10.4
https://pub.dev/packages/youtube_explode_dart/changelog

Reply:
but the sdk was not supporting

Reply:
[raag] flutter pub get
Running "flutter pub get" in raag...                            
Because every version of flutter from sdk depends on meta 1.3.0 and json_annotation >=4.1.0 depends on meta ^1.4.0, flutter from sdk is incompatible with json_annotation >=4.1.0.

And because youtube_explode_dart >=1.10.0 depends on freezed_annotation ^0.14.3 which depends on json_annotation ^4.1.0, flutter from sdk is incompatible with youtube_explode_dart >=1.10.0.

So, because raag depends on both youtube_explode_dart ^1.10.4 and flutter any from sdk, version solving failed.
pub get failed (1; So, because raag depends on both youtube_explode_dart ^1.10.4 and flutter any from sdk, version solving failed.)
exit code 1

Reply:
Oh okay